src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The interface name `FruitList` is not descriptive of the behavior it's supposed to represent. Consider renaming it to something more indicative of its purpose, like `FruitTransactionProcessor`.